spanner/tests/system/test_system.py

```diff
@@ -395,6 +432,53 @@ def _unit_of_work(transaction, name):
         self.assertEqual(len(rows), 2)
 
 
+    def test_multi_processing_transaction(self):
+        retry = RetryErrors(exceptions.ServiceUnavailable)
+
+        from multiprocessing import Pool
+        processes = 3
+        pool = Pool(processes)
```

Comment:
I suspect if you drop this below `client = Client()` in 442 this will fail.

Reply:
still passing after moving the pool below client.

Reply:
If you make a call before constructing the pool does it still work?

Reply:
I moved the pool construction to after line 463 and it still worked.

Reply:
hrm

Reply:
Would I need to update some pip files or would nox take care of that?

Reply:
nox should handle that, but it might be useful to try reproducing this outside of our system testing harness.

Reply:
`_process` creates its own client at 66. So the client being created here should not be relevant.

Reply:
@vkedia that's what one would think, but according to the gRPC team interacting with gRPC at all in the process starts persistent state (threads) that does not survive `fork`.

Reply:
I see. So even if they create a new Client object in the subprocess (which is what is happening here), it is not supposed to work?
What I am trying to say is, even though a Client object is being created here, it is not being used in the sub processes since they are all creating there own client objects. IIUC you are saying even that should not work?

Reply:
From my understanding that is correct.